### ISSUE TITLE
error

### ISSUE THREAD

**Issue by EddieMorra** [avocode/avocode#1893]
---
* **Version**: 2.19.2
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 274933
---
```
TypeError: Cannot read property 'toJS' of null
    at t._addDocumentLayerSliceLayer (file:///C:/Users/roma/AppData/Local/avocode/app-2.19.2/resources/app.asar/dist/avocode.min.js:1:3071400)
    at t._handleAction (file:///C:/Users/roma/AppData/Local/avocode/app-2.19.2/resources/app.asar/dist/avocode.min.js:1:3069166)
    at Object.ID_85 (file:///C:/Users/roma/AppData/Local/avocode/app-2.19.2/resources/app.asar/dist/avocode.min.js:1:3066587)
    at Dispatcher._invokeCallback (C:\Users\roma\AppData\Local\avocode\app-2.19.2\resources\app.asar\node_modules\flux\lib\Dispatcher.js:198:24)
    at Dispatcher.dispatch (C:\Users\roma\AppData\Local\avocode\app-2.19.2\resources\app.asar\node_modules\flux\lib\Dispatcher.js:174:14)
    at Dispatcher.dispatch (C:\Users\roma\AppData\Local\avocode\app-2.19.2\resources\app.asar\node_modules\@avocode\flux\lib\dispatcher.js:76:28)
    at e.value (file:///C:/Users/roma/AppData/Local/avocode/app-2.19.2/resources/app.asar/dist/avocode.min.js:1:3055522)
    a
```

**Comment by vojtatranta** [comment]
Hi @EddieMorra ,
this is a bug for sure.

It seems that you were trying to create a slice, right?

**Comment by EddieMorra** [comment]
yep

**Comment by vojtatranta** [comment]
Aha and any other slice it is possible to create? Or are you now stuck?

**Comment by EddieMorra** [comment]
possible, but when i select some block with slice and click on white empty space i have this error
, not critical, but thx for reply

**Comment by vojtatranta** [comment]
What do you mean by `white empty space`? Can you send gif or video please?

**Comment by EddieMorra** [comment]
![bandicam_2018-02-20_16-31-35-577](https://user-images.githubusercontent.com/17853741/36430063-ec04416c-165c-11e8-87f6-72619a9e76ec.gif)

**Comment by vojtatranta** [comment]
This should be fixed and released in `3.0.0`
